frontend/src/component/admin/network/NetworkOverview/NetworkOverview.tsx

```diff
@@ -84,37 +115,71 @@ const toGraphData = (metrics?: RequestsPerSecondSchema) => {
 
 export const NetworkOverview = () => {
     usePageTitle('Network - Overview');
-    const { themeMode } = useThemeMode();
+    const theme = useTheme();
     const { metrics } = useInstanceMetrics();
     const apps = useMemo(() => {
         return toGraphData(metrics);
     }, [metrics]);
 
-    const graph = `
-    graph TD
-        subgraph _[ ]
-        direction BT
-            Unleash(<img src='${formatAssetPath(
-                themeMode === 'dark' ? logoWhiteIcon : logoIcon,
-            )}' width='72' height='72' class='unleash-logo'/><br/>Unleash)
-            ${apps
-                .map(
-                    ({ label, reqs, type }, i) =>
-                        `app-${i}("${label.replaceAll(
-                            '"',
-                            '&quot;',
-                        )}") -- ${reqs} req/s<br>${type} --> Unleash`,
-                )
-                .join('\n')}
-        end
-    `;
+    if (apps.length === 0) {
+        return <Alert severity='warning'>No data available.</Alert>;
+    }
+
+    const mockApps = new Array(30).fill(1).map((_, i) => ({
```

Comment:
is the mock apps intended?

Reply:
Oops, you're right. I thought I had reverted this but it slipped by. Nice catch! Cleaned up in https://github.com/Unleash/unleash/pull/6320/commits/b2221f7d96b959b9902a5687750b49a2c1696cc4